src/Integration/Rules/QualityProfile.cs

```diff
@@ -0,0 +1,36 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+
+namespace SonarLint.VisualStudio.Integration.Rules
+{
+    public class QualityProfile
+    {
+        public QualityProfile(Language language, IEnumerable<SonarRule> rules)
+        {
+            Language = language;
```

Comment:
Does the existing code that fetches the QP return the QP name? If so, it would be useful to have it on the interface e.g. for displaying in UI / logging.

Reply:
I just think we don't need anything more until we have a first (working) version we can test.